foxpuppet/region.py

```diff
@@ -0,0 +1,21 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+class Region(object):
+    """A region object.
+
+    Used as a base class for region objects.
+
+    :param window: Window object this region appears in.
+    :param root: element that serves as the root for the region.
+    :type window: :py:class:`~.windows.BaseWindow`
+    :type root: :py:class:`~selenium.webdriver.remote.webelement.WebElement`
+    """
```

Comment:
I assume the above docstring is for the class properties. Can you add the same for the __init__ method please?